File: 3685-count-subarrays-of-length-three-with-a-condition/3685-count-subarrays-of-length-three-with-a-condition.py
class Solution:
    def countSubarrays(self, nums: List[int]) -> int:
        c=0
        for i in range(len(nums)):
            for j in range(i+2,len(nums)):
                if(nums[i]+nums[j]==float(nums[i+1]/2)):
                    c+=1
                    break
                break
        return c

        

File: 3685-count-subarrays-of-length-three-with-a-condition/3685-count-subarrays-of-length-three-with-a-condition.cpp
# include <cmath>
class Solution {
public:
    int countSubarrays(vector<int>& nums) {
        int c = 0;
        for (int i = 0; i < nums.size(); i++) {
            for (int j = i + 2; j < nums.size(); j++) {
                if (nums[i] + nums[j] == double(nums[i + 1]) / 2) {
                    c++;
                    break;
                }
                break;
            }
        }
        return c;
    }
};